rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -722,7 +723,7 @@ type ExerciseTerms: <"A class defining the exercise period for an option togethe
     earliestExerciseTime BusinessCenterTime (0..1) <"The earliest time at which notice of exercise can be given by the buyer to the seller (or seller's agent) to, and including, the expiration date.">
     latestExerciseTime BusinessCenterTime (0..1) <"For a Bermuda or American style option, the latest time on an exercise business day (excluding the expiration date) within the exercise period that notice can be given by the buyer to the seller or seller's agent. Notice of exercise given after this time will be deemed to have been given on the next exercise business day.">
     expirationTime BusinessCenterTime (1..1) <"The latest time for exercise on expirationDate. It is made mandatory given that for all option styles, this field is required.">
```

Comment:
The `expirationTime` attribute needs to be made optional, and the condition over its absence or existence in relation to `expirationTimeType` be reinstated